circle.yml

```diff
@@ -0,0 +1,21 @@
+machine:
+  xcode:
+    version: "8.0"
+
+dependencies:
+  pre:
```

Comment:
#req I believe we need a `bundle install` and, depending on how Circle is set up, a `gem install bundler`